source/UF/SizeTruncationandConnectedness.lagda

```diff
@@ -0,0 +1,346 @@
+Ian Ray, 4th Febuary 2024.
+
+Modifications made by Ian Ray on 14 October 2024.
+
+Modifications made by Ian Ray on 7 January 2025.
+
+We develop some results that relate size, truncation and connectedness from
+the following paper:
+Christensen, J.D. (2024), Non-accessible localizations.
```

Comment:
`[1] Christensen, J.D. (2024), Non-accessible localizations.` 

So that you can refer to it below as `[1]`.